registry-requirements.md

```diff
@@ -120,7 +120,19 @@ What does the binding have to contain to go into the table
     - Implementing a non-commercial Consumer application/driver
     - Conditions for commercial use: E.g. building a commercial product with the binding
     - Making a statement about your product's supporting that binding
-9. DISCUSS: What is the objective mechanism to confirm the status change from "initial" to "current"? Should we have a test suite? Plugfest experience?
+9. Transition from "Initial" to "Current"
+  - Starting from the initial submission each binding has to demonstrate a certain level of concrete development maturity. This process involves real-world testing, which can take place in Plugfests, independent testing events, or even informal collaboration between developers. These testing events do not have to be organized by W3C and can be conducted remotely, including over VPN. The goal is to demonstrate that the binding correctly maps protocol operations and is well understood by at least two parties.
+  - At each testing event every operation defined in the binding MUST be validated automatically (e.g., scripts, test suites, etc.) and the results SHOULD be published in a dedicated document (README, or other human-readable documents) called *Test Report*.
+  - A *Test Report*
+      - MUST contain information on the testing environment
+      - MUST provide an example of the logical process (not necessarily code) about how a TD can be processed to establish a communication between consumer and exposer.
```

Comment:
This was intended to be actually more like a use case, e.g. controlling temperature in a room by sending requests. However, what you are requiring also makes sense

Reply:
TD Call: this makes more sense as part of the binding itself as it does not change per implementation

Reply:
```suggestion
      - MUST provide an example of the logical process (not necessarily code) about how a TD can be processed to establish a communication between consumer and exposer.
      - MUST contain information about the scenario that was tested, e.g. controlling the room temperature by measuring temperature and adjusting the heater.
```